spec/ttfunk/table/cff/charset_spec.rb

```diff
@@ -14,7 +14,7 @@
 
       it 'includes all the entries from the expert charset' do
         expect(charset.to_a).to eq(
-          ['.notdef'] + described_class.strings_for_charset_id(charset_id)
+          ['.notdef'] + described_class.strings_for_charset_id(charset_id).to_a
```

Comment:
You can spare explicit `to_a` if you implement `to_ary` method on `OneBasedArray`.

Example:
```ruby
class A
  def to_ary
    [:A]
  end
end

[1] + A.new # => [1, :A]
```

Reply:
Wow I didn't know about that trick, thanks!